frontend/src/lib/Status.tsx

```diff
@@ -10,6 +10,9 @@ export const RouteOnNotOK = async (response: Response, navigate: NavigateFunctio
     case 404:
       navigate('/notFound');
       break;
+    case 500:
```

Comment:
<img width="1361" alt="image" src="https://github.com/jykuo-love-shiritori/twp/assets/92158595/f6943b53-cb30-46ab-bc79-2106f7b77560">
because this error is not right